sdk/communication/communication-chat/src/models/mappers.ts

```diff
@@ -172,12 +173,25 @@ export const mapToChatContentSdkModel = (
   return result;
 };
 
+/**
+ * @internal
+ */
+export const mapToChatPolicyViolationSdkModel = (
+  policyVolation: RestModel.PolicyViolation,
+): PolicyViolation => {
+  const { state } = policyVolation;
+  const result: PolicyViolation = { 
+    result: state
+  };
```

Comment:
This is pretty confused in reading after the renaming

Reply:
Other developer would be confused about the difference between `state` and `result` here.

Reply:
Resolved as per conversation with ARB